openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -828,6 +828,12 @@ The `c_nonce` value can be retrieved from the Nonce Endpoint as defined in (#non
 Additional Credential Request parameters MAY be defined and used.
 The Credential Issuer MUST ignore any unrecognized parameters.
 
+The Credential Issuer indicates support for encrypted requests by including the `credential_request_encryption` parameter in the Credential Issuer Metadata. The Client MAY encrypt the request when `encryption_required` is `false` and MUST do so when `encryption_required` is `true`. 
```

Comment:
If request encryption is bound to response encryption, we should only have a single issuer metadata parameter?

Reply:
Discussed on WG call: This would tie the two features together, we may make changes in the future that make you can use response encryption without request encryption, so better to keep them separate.